frontend/src/components/AssignmentModal/ContactInfoCard.vue

```diff
@@ -0,0 +1,418 @@
+<template>
+  <v-card-text>
+    <v-container 
+      v-for="contactData in assignmentState.contactsInfo"
+      :key="contactData.id"
+    >
+      <v-row v-if="!contactData.id">
+        <v-col
+          cols="12"
+        >
+          <p class="text-subtitle-1">{{ contactData.type }}</p>
+        </v-col>
+      </v-row>
+      <v-row v-else>
+        <v-col
+          cols="12"
+          sm="11"
+        >
+          <CustomSelect 
+            v-model="contactData.type"
+            :items="getContactTypesLabels"
+            :label="'Contact type*'"
+          />
+        </v-col>
+        <v-col
+          v-if="contactData.id"
+          cols="12"
+          sm="1"
+        >
+          <v-btn
+            style="transform: translateY(50%);"

```

Comment:
you add this style property a few times, better to create a CSS class for this purpose